packages/smart-router/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@pancakeswap/smart-router",
+  "version": "0.0.1",
+  "description": "A SDK for getting best routes from AMM",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pancakeswap/pancake-frontend.git",
+    "directory": "packages/smart-router"
+  },
+  "scripts": {
+    "build": "tsup",
+    "dev": "tsup --watch",
+    "prepublishOnly": "yarn run build",
+    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist"
+  },
+  "dependencies": {
+    "@pancakeswap/farms": "*",
+    "@pancakeswap/multicall": "*",
+    "@pancakeswap/sdk": "*",
+    "@pancakeswap/tokens": "*",
+    "@pancakeswap/token-lists": "*",
+    "ethers": "^5.7.2",
```

Comment:
might need to fix the version, better split `ethers`

Reply:
OK.

Reply:
Done